### DISCUSSION TITLE
React Native Vector Icons don't show after packed

### DISCUSSION THREAD

**Discussion by leezaric** [expo/expo#27382] [Archived: Questions]
I'm using expo to develop react native app. 
When I debug using "**npx expo start --web**", the vector icons do work in the browser.
![image](https://github.com/expo/expo/assets/40030561/62d6884c-90d7-4586-a2b4-1938f6532225)
But after packing the project using "**npx expo export --platform web**" and deployed to Firebase, the icons can't show on web. 
![image](https://github.com/expo/expo/assets/40030561/7196e561-6fcf-4b17-9523-304da7864299)

Could anyone provide some tips, thanks.

**Comment by vinaciotm** [comment]
same here. after npx vercel of folder /dist and check online has no icons, but random shapes.

**Comment by deepakkumardk** [comment]
Facing the same issue on my project (https://github.com/deepakkumardk/react-native-blossom-ui/tree/main/apps/native) too!!
If anyone has any fix, would love to hear that!
